src/App.tsx

```diff
@@ -69,7 +68,6 @@ export function App() {
               <Route path="/auth/verify" component={AuthVerify} />
               <Route path="/register" component={Register} />
               <Route path="/contact" component={BraveAdsContactFrame} />
-              <Route path="/search" component={SearchLandingPage} />
```

Comment:
Are we breaking this url?  Once we've put something on a url, we should never remove it, at very least redirect it.

Reply:
Yes we absolutely need not to break this url - this is a core principle of web development. Once a URL serves content publicly on the internet, it usually gets remembered.

In this case, looking at matomo I'm not seeing any visits. So please just redirect to the home page - currently it redirects to `/auth/link` which is not good.